commands/command.md

```diff
@@ -1,52 +1,79 @@
 Returns @array-reply of details about all Redis commands.
 
-Cluster clients must be aware of key positions in commands so commands can go to matching instances,
-but Redis commands vary between accepting one key,
-multiple keys, or even multiple keys separated by other data.
+Cluster clients must be aware of key positions in commands so commands can go to matching instances, but Redis commands vary between accepting one key, multiple keys, or even multiple keys separated by other data.
+
+Starting from Redis 7.0.0 a more flexible mechanism was introduced in order to help clients finding the key positions.
+For more information please check the [key-specs page][tr].
+[tr]: /topics/key-specs
 
 You can use `COMMAND` to cache a mapping between commands and key positions for
 each command to enable exact routing of commands to cluster instances.
 
-## Nested Result Array
-Each top-level result contains six nested results.  Each nested result is:
-
-  - command name
-  - command arity specification
-  - nested @array-reply of command flags
-  - position of first key in argument list
-  - position of last key in argument list
-  - step count for locating repeating keys
+`COMMAND` has several subcommands:
+
+ - `COMMAND INFO`
```

Comment:
COMMAND INFO was improved in 7.0 to be able to take 0 arguments and behave the same as COMMAND COMMAND.
i guess this PR should update command-info.md

Reply:
nothing in command-info.md says that COMMAND INFO takes at least one command + the recent change appears in the history section

I guess that's enough, no?

Reply:
ok